src/app/notice/page.tsx

```
@@ -0,0 +1,25 @@
+        return (
+          <>
+            <NoticeItem
+              key={id}
+              id={id}
+              title={title}
+              clipType={clipType as "이벤트" | "공지" | "매거진"}
+              createdAt={createdAt}
+            />
+            <div className="h-[1px] w-full bg-grey" />
+          </>
+        );
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)